README.markdown

```diff
@@ -144,6 +144,18 @@ npm install --global @sourcemeta/jsonschema
 pip install sourcemeta-jsonschema
 ```
 
+### From mise
+
+```sh
+mise use jsonschema
```

Comment:
Interesting. I never hears about these package managers. Are they specifically integrating with this tool? Might be worth putting a link to their homepages and the manifests that they have created for this tool, if any?